administrator/components/com_cronjobs/src/Model/CronjobsModel.php

```diff
@@ -0,0 +1,378 @@
+	 * @since  __DEPLOY_VERSION__
+	 */
+	protected function getListQuery(): QueryInterface
+	{
+		// Create a new query object.
+		$db = $this->getDbo();
+		$query = $db->getQuery(true);
+		$user = Factory::getApplication()->getIdentity();
+
+		/*
+		 * Select the required fields from the table.
+		 * ? Do we need all these defaults ?
+		 * ? Does 'list.select' exist ?
+		 */
+		$query->select(
+			$this->getState(
+				'list.select',
+				'a.id, a.asset_id, a.title, a.type, a.trigger, a.execution_rules, a.state, a.last_exit_code' .
+				', a.last_execution, a.next_execution, a.times_executed, a.times_failed, a.ordering, a.note'
+			)
+		);
+
+		// From the #__cronjobs table as 'a'
+		$query->from($db->quoteName('#__cronjobs', 'a'));
+
+		// Filters go below
+		$filterCount = 0;
+
+		/**
+		 * Extends query if already filtered.
+		 *
+		 * @param   string  $outerGlue
+		 * @param   array   $conditions
+		 * @param   string  $innerGlue
+		 *
+		 * @since __DEPLOY_VERSION__
+		 */
+		$extendIfFiltered = function (
+			string $outerGlue, array $conditions, string $innerGlue
+		) use ($query, &$filterCount) {
+			if ($filterCount)
+			{
+				$query->extendWhere($outerGlue, $conditions, $innerGlue);
+			}
+			else
+			{
+				$query->where($conditions, $innerGlue);
+			}
+
+		};
+
+		// Filter over state ----
+		$state = $this->getState('filter.state');
+
+		if (is_numeric($state))
+		{
+			$filterCount++;
+			$state = (int) $state;
+			$query->where($db->quoteName('a.state') . '= :state')
+				->bind(':state', $state);
+		}
+
+		// Filter over type ----
+		if ($typeFilter = $this->getState('filter.type'))
+		{
+			$filterCount++;
+			$query->where($db->quotename('a.type') . '= :type')
+				->bind(':type', $typeFilter);
+		}
+
+		// TODO: Filter over trigger
+
+		// Filter over exit code ----
+		$exitCode = $this->getState('filter.last_exit_code');
+
+		if (is_numeric($exitCode))
+		{
+			$filterCount++;
+			$exitCode = (int) $exitCode;
+			$query->where($db->quoteName('a.last_exit_code') . '= :last_exit_code')
+				->bind(':last_exit_code', $exitCode, ParameterType::INTEGER);
+		}
+
+		// Filter due ----
+		if (is_numeric($due = $this->getState('filter.due')))
+		{
+			$now = Factory::getDate('now', 'GMT')->toSql();
+			$operator = $due === 1 ? '<= ' : '> ';
+			$filterCount++;
+			$query->where($db->qn('a.next_execution') . $operator . ':now')
+				->bind(':now', $now);
+		}
+
+		// Filter over search string if set (title, type title, note, id) ----
+		$searchStr = $this->getState('filter.search');
+
+		if (!empty($searchStr))
+		{
+			// Allow search by ID
+			if (stripos($searchStr, 'id:') === 0)
+			{
+				// Add array support [?]
+				$id = (int) substr($searchStr, 3);
+				$query->where($db->quoteName('a.id') . '= :id')
+					->bind(':id', $id, ParameterType::INTEGER);
+			}
+			// Search by type is handled exceptionally in _getList() [TODO: remove refs]
+			elseif (stripos($searchStr, 'type:') !== 0)
+			{
+				$searchStr = "%${searchStr}%";
```

Comment:
You dont need curly brackets if your are using double quotes around your string. 

```suggestion
				$searchStr = "%$searchStr%";
```